pkg/bucketaws/setup.go

```diff
@@ -18,64 +18,131 @@ package s3setup
 import (
 	"fmt"
 	"os"
+	"strings"
+	"time"
+
+	files "github.com/redhatcre/syncron/utils"
+
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 )
 
+func ProcessDate(fromDate time.Time) []string {
+
+	// This function formats all dates from provided date
+	// until current date and returns a string formatted
+	// for s3 keys.
```

Comment:
Documentation in Go goes on top of the function I believe. Maybe not.

Reply:
You are right, fixed.